gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -162,13 +162,14 @@ def _bigquery_to_vcf_shards(
   Also, it writes the meta info and data header with the call names to
   `vcf_header_file_path`.
   """
+  schema = _get_schema(known_args.input_table)
+  columns = _get_query_columns(schema)
```

Comment:
nit: I think it's better to pass the `schema` to `_get_bigquery_query` and get the columns inside that method (unless you're planning to reuse columns somewhere else later). The main reason is to make the API of `_get_bigquery_query` more generic and also limiting the scope of `columns` to the only method that actually needs it.